trapdata/api/models/localization.py

```
@@ -56,3 +59,64 @@ def save_detection(image_id, coords):
+            bbox = BoundingBox(x1=coords[0], y1=coords[1], x2=coords[2], y2=coords[3])
+            detection = DetectionResponse(
+                source_image_id=image_id,
+                bbox=bbox,
+                inference_time=seconds_per_item,
+                algorithm=AlgorithmReference(name=self.name, key=self.get_key()),
+                timestamp=datetime.datetime.now(),
+                crop_image_url=None,
+            )
+            return detection
+
+        for image_id, image_output in zip(item_ids, batch_output):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)